README.md

```diff
@@ -5,106 +5,108 @@
+f,      U+0066,     base16,             hexadecimal,                                                    final
+F,      U+0006,     base16upper,        hexadecimal,                                                    final
+v,      U+0076,     base32hex,          rfc4648 case-insensitive - no padding - highest char,           experimental
+V,      U+0056,     base32hexupper,     rfc4648 case-insensitive - no padding - highest char,           experimental
+t,      U+0074,     base32hexpad,       rfc4648 case-insensitive - with padding,                        experimental
+T,      U+0054,     base32hexpadupper,  rfc4648 case-insensitive - with padding,                        experimental
+b,      U+0062,     base32,             rfc4648 case-insensitive - no padding,                          final
+B,      U+0042,     base32upper,        rfc4648 case-insensitive - no padding,                          final
+c,      U+0063,     base32pad,          rfc4648 case-insensitive - with padding,                        draft
+C,      U+0043,     base32padupper,     rfc4648 case-insensitive - with padding,                        draft
+h,      U+0068,     base32z,            z-base-32 (used by Tahoe-LAFS),                                 draft
+k,      U+006b,     base36,             base36 [0-9a-z] case-insensitive - no padding,                  draft
+K,      U+004b,     base36upper,        base36 [0-9a-z] case-insensitive - no padding,                  draft
+z,      U+007a,     base58btc,          base58 bitcoin,                                                 final
+Z,      U+005a,     base58flickr,       base58 flicker,                                                 experimental
+m,      U+006d,     base64,             rfc4648 no padding,                                             final
+M,      U+004d,     base64pad,          rfc4648 with padding - MIME encoding,                           experimental
+u,      U+0075,     base64url,          rfc4648 no padding,                                             final
+U,      U+0055,     base64urlpad,       rfc4648 with padding,                                           final
+p,      U+0070,     proquint,           [PRO-QUINT],                                                    experimental
+🚀,     U+1F680,    base256emoji,       base256 with custom alphabet using variable-sized-codepoints,   experimental
+
 ```
 
-**NOTE:** Multibase-prefixes are encoding agnostic. "z" is "z", not 0x7a ("z" encoded as ASCII/UTF-8). For example, in UTF-32, "z" would be `[0x7a, 0x00, 0x00, 0x00]`.
+**NOTE:** Multibase-prefixes are encoding agnostic and their canonical form is a Unicode [code point], not an ASCII character or corresponding UTF-8 bytes.  
+Since UTF-8 is the most common context for binary data that gets prefixed as a multibase today, the UTF-8 column is provided as a reference for detecting multibase-prefixes, since most of these codes can be detected in the first byte in known-encoding contexts. 
```

Comment:
There is no UTF-8 column anymore. Though if we rename "code" to "ASCII", then this paragraph would still be accurate when "UTF-8" is changed to "ASCII:.